lib/ansible/modules/database/mysql/mysql_db.py

```diff
@@ -104,6 +104,13 @@
     choices: [0, 1, 2]
     default: 0
     version_added: '2.10'
+  args:
```

Comment:
I would rename this to something like `dump_extra_args` or even `mysqldump_extra_args`. The operation should be included here IMO, and also `extra` is kind of telling since this aren't the *only* options passed to `mysqldump`.

Reply:
dump_extra_args LGTM